src/components/Sessions/RequireSession.js

```diff
@@ -1,42 +1,77 @@
-import React from 'react';
+import React, {useEffect, useReducer} from 'react';
 import PropTypes from 'prop-types';
 import {Link} from 'react-router-dom';
 import {FormattedMessage} from 'react-intl';
 
 import Anchor from 'components/Anchor';
 import Card from 'components/Card';
 import ErrorMessage from 'components/ErrorMessage';
+import Modal from "components/modals/Modal";
 
-// import SessionExpiry from './SessionExpiry';
```

Comment:
code is not used anywhere and you removed the last reference to the comment, so that component can be deleted alltogether